tests/gdx-tests/src/com/badlogic/gdx/tests/FilesTest.java

```diff
@@ -208,11 +208,7 @@ private void testInternal () throws IOException {
 			fail();
 		} catch (Exception ignored) {
 		}
-		FileHandle dir;
-		if (Gdx.app.getType() == ApplicationType.Android || Gdx.app.getType() == ApplicationType.iOS)
-			dir = Gdx.files.internal("data");
-		else
-			dir = Gdx.files.internal("../gdx-tests-android/assets/data");
+		FileHandle dir = Gdx.files.internal("data");

```

Comment:
Is this an intended change? Looks unrelated to the topic.

Reply:
That the test was broken and I ended up fixing it while checking if the update didn't introduce any new issue. Should I take this out?

Reply:
Its preferred if you split up unrelated issues/pull requests. Or at very least if they are tiny changes document what you have changed and why.

Reply:
We'll make an exception just for you this time  ;]